src/api/unsplash-api.ts

```diff
@@ -0,0 +1,36 @@
+import { QueryFunction } from '@tanstack/react-query';
+// import { SearchParamsValues } from 'context/SearchParamsContext';
+import { Image, SearchResponse } from 'types/unsplash-types';
+import { ColorType, ContentFilterType, OrderByType, OrientationType } from 'types/unsplash-types';
+
+const baseUrl = 'https://api.unsplash.com';
+
+export const getPhotos: QueryFunction<SearchResponse | Image[], string[], number> = async ({
+  pageParam,
+}) => {
+  const initParams = {
```

Comment:
u should pass this data as arguments instead of get it from localStorage directly

Reply:
rewrited